indigo_app/templates/indigo_api/_work_repeal_form.html

```diff
@@ -0,0 +1,59 @@
+{% load i18n %}
+
+<div class="form-group row">
+  <label class="col-2 col-form-label">{% trans 'Repealed by' %}</label>
+  <div class="col">
+    <input type="hidden" id="{{ form.repealed_by.id_for_label }}" name="{{ form.repealed_by.html_name }}" value="{{ form.repealed_by.value|default:'' }}">
+    {% if work.repealed_by %}
+      <div class="mb-2">
+        <a href="{% url 'work' work.repealed_by.frbr_uri %}">{{ work.repealed_by.title }}</a>
+        <br>
+        <span class="text-muted">{{ work.repealed_by.frbr_uri }}</span>
+      </div>
+    {% endif %}
+
+    <div>
+      {% url 'work_edit_repeal' work.frbr_uri as edit_url %}
+      <button
+        class="btn btn-outline-primary"
+        type="button"
+        hx-get="{% url 'place_work_chooser' place.place_code %}?submit={{ edit_url|urlencode }}&target={{ "#work-form-repeal"|urlencode }}&field=repealed_by"
```

Comment:
This loads the modal content when the button is clicked and passes in parameters that are used when the work is chosen, and the OK button is clicked:

* submit: the hx-get URL that will be used when the work is chosen
* target: the hx-target that will be used when the work is chosen